File: itsm/src/main/kotlin/com/brainz/workflow/process/ProcessManager.java
package com.brainz.workflow.process;

import java.util.List;

public class ProcessManager {

    private List<Object> accessList;
    
    public ProcessManager createQuery() {
        accessList.clear();
        return this;
    }

    public ProcessManager accessGroup(String group) {
        accessList.add(group);
        return this;
    }

    public ProcessManager accessAsignee(String asignee) {
        accessList.add(asignee);
        return this;
    }
    
//    public List<Process> list() {
//        List<Process> processList = new List<Process>();
//        { new Process("test1"), new Process("test2") };
//            
//    }

}

File: itsm/src/main/kotlin/co/brainz/workflow/process/ProcessManager.kt
package co.brainz.workflow.process

public class ProcessManager {
	
	private lateinit var accessList : MutableList<Any>
	
	fun createQuery() : ProcessManager {
		accessList.clear();
		return this;
	}
	
	fun accessGroup(group : String) : ProcessManager {
		accessList.add(group)
		return this;
	}
	
	fun accessAsignee(asignee : String) : ProcessManager {
		accessList.add(asignee)
		return this;
	}
}
